packages/experimental/kai/src/frames/Note/NoteFrame.tsx

```diff
@@ -42,17 +66,24 @@ export const TileContent = withReactor(({ item, selected, onDelete }: TileConten
     <div className='flex flex-1 flex-col overflow-hidden'>
       <div className='flex w-full items-center mb-3'>
         {/* Title */}
-        <div className='flex flex-1 overflow-hidden'>
-          <input
-            ref={inputRef}
-            className='text-lg w-full outline-0 p-1 bg-transparent'
+        <div className='flex flex-col flex-1 overflow-hidden'>
+          <Input
+            label='Title'
+            variant='subdued'
+            value={note.title}
+            onChange={(event) => {
+              note.title = event.target.value;
+            }}
             placeholder='Title'
-            autoFocus={selected} // TODO(burdon): Not working.
-            value={item.label}
-            onChange={({ target: { value } }) => {
-              // TODO(burdon): Breaks abstraction: bubble change event instead.
-              const note = item.data! as Note;
-              note.title = value;
+            slots={{
+              label: { className: 'sr-only' },
```

Comment:
I keep forgetting, but if the `label` slot is just the `sr-only` classname, you can instead use the boolean prop `labelVisuallyHidden` which does this for you.

Reply:
is the label useful for any reason (e.g., documentation) in SR?